plugin/src/main/scala/tut/Plugin.scala

```diff
@@ -73,13 +72,15 @@ object Plugin extends sbt.Plugin {
       libraryDependencies += "org.tpolecat" %% "tut-core" % BuildInfo.version % "test",
       tutSourceDirectory := sourceDirectory.value / "main" / "tut",
       tutTargetDirectory := crossTarget.value / "tut",
-      watchSources <++= tutSourceDirectory map { path => (path ** "*.md").get },
       tutScalacOptions := {
         val testOptions = scalacOptions.in(Test).value
         val unwantedOptions = Set("-Ywarn-unused-import")
         testOptions.filterNot(unwantedOptions)
       },
       tutNameFilter := """.*\.(md|markdown|txt|htm|html)""".r,
+      watchSources in Defaults.ConfigGlobal ++= (tutSourceDirectory.value ** new NameFilter {
```

Comment:
Is there any particular reason that this switched from `watchSources` to `watchSources in Defaults.ConfigGlobal`? Could that potentially have any unintended consequences?